server/src/main/java/org/activityinfo/ui/client/page/entry/sitehistory/SiteHistoryRenderer.java

```diff
@@ -62,6 +62,11 @@ public String render(ActivityFormDTO activityForm, List<LocationDTO> locations,
 
         boolean first = true;
         for (SiteHistoryDTO history : histories) {
+
+            if (isAttachmentHistoryEntry(history.getEntityName())) {
```

Comment:
Isn't this skipping _all_ fields in an update if the update contains an attachment? If for example, I update a site, changing a quantity field _and_ an attachment, won't that skip both changes here? It seems like you should be just checking the indicator type of the field.